src/services/image-snapshot-service.ts

```diff
@@ -92,6 +93,7 @@ export default class ImageSnapshotService extends PercyClientService {
       name,
       widths: [width],
       minimumHeight: height,
+      clientInfo: `percy-upload/${pkg.version}`,
```

Comment:
Is the package version necessary since we include the version of agent?

Reply:
Probably not. 🤷‍♂

Reply:
Removed the version since it's redundant

Reply:
Removed the version since it's redundant